javascript-local-storage/data.js

```diff
@@ -0,0 +1,8 @@
+/* exported todos */
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
If you try this in an incognito window (private browsing) it will break. Be sure to use a conditional to test for `null`.